webdriver/src/com/googlecode/jmeter/plugins/webdriver/sampler/gui/WebDriverSamplerGui.java

```diff
@@ -1,35 +1,30 @@
 package com.googlecode.jmeter.plugins.webdriver.sampler.gui;
 
 import com.googlecode.jmeter.plugins.webdriver.sampler.WebDriverSampler;
-import jsyntaxpane.DefaultSyntaxKit;
 import kg.apc.jmeter.JMeterPluginsUtils;
 import org.apache.jmeter.samplers.gui.AbstractSamplerGui;
 import org.apache.jmeter.testelement.TestElement;
 import org.apache.jmeter.util.JSR223BeanInfoSupport;
 import org.apache.jorphan.logging.LoggingManager;
 import org.apache.log.Logger;
-
+import org.fife.ui.rsyntaxtextarea.RSyntaxTextArea;
+import org.fife.ui.rtextarea.RTextScrollPane;
 import javax.swing.*;
 import java.awt.*;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
+import org.apache.jmeter.gui.util.JSyntaxTextArea;
+import org.apache.jmeter.gui.util.JTextScrollPane;
 
 public class WebDriverSamplerGui extends AbstractSamplerGui {
 
     private static final long serialVersionUID = 100L;
     private static final Logger LOGGER = LoggingManager.getLoggerForClass();
 
-    static {
```

Comment:
I changed "text area" of web driver's script. I did it because the default JComponent acting as a text area for script (actually I forgot it's name) did not deal with changing language of script. For example, if I chose groovy, java it throws exception and display a dialog any time I focus the editor. And It can't fold code, can't colorize the text (groovy or java I mean). So I used a component as JSR223 Sampler did - JSyntaxTextArea. I think it is greatest swing text editor component I even seen. So why to not use it? 
So I erased that static initializer.

May be you have any reasons to leave original text editor component?